guides/source/active_storage_overview.md

```diff
@@ -539,60 +539,76 @@ Analysis requires the `mini_magick` gem. Video analysis also requires the [FFmpe
 
 [`analyzed?`]: https://api.rubyonrails.org/classes/ActiveStorage/Blob/Analyzable.html#method-i-analyzed-3F
 
-Transforming Images
--------------------
+Displaying Images, Videos, and PDFs
 
-To enable variants, add the `image_processing` gem to your `Gemfile`:
+Active Storage supports [representing][] a variety of files. You can use the `representation` method on an attachment to display an image variant, or a preview of a video or PDF. Before calling `representation`, check if the attachment can be represented by calling `representable?`. Some file formats can't be previewed natively in the browser (eg. Word documents); if `representable?` returns false you may want to [link to](#linking-to-files) the file instead.
+
+```erb
+<ul>
+  <% @message.files.each do |file| %>
+    <li>
+      <% if file.representable? %>
+        <%= image_tag file.representation(resize_to_limit: [100, 100]) %>
+      <% else %>
+        <%= link_to url_for(file) do %>
+          Download file
+        <% end %>
```

Comment:
Instead of linking to the file, what about displaying a placeholder image (e.g. `image_tag "placeholder.png"`), for uniformity?

Reply:
Nice. I might do both.